lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
+
+use strict;
+use DDG::Spice;
+
+name 'PHL Indego Bikeshare Locations';
+source 'Indego Philadelphia';
+icon_url 'http://www.rideindego.com/';
+description 'Search Indego Bike stations in Philadelphia';
+primary_example_queries 'indego philadelphia', 'bikeshare philadelphia';
+secondary_example_queries 'philly bikeshare';
+category 'geography';
+topics 'everyday', 'travel', 'entertainment', 'geography';
+code_url 'https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/IndegoBikeshareLocations.pm';
+attribution github => 'marianosimone',
+            github => 'AcriCAA', 
+            web => ['http://www.marianosimone.com',  'Mariano Simone'],
+            web  => ['http://www.coreyacri.com',  'Corey Acri'],
+            web => ['https://codeforphilly.org/', "Built at Code for Philly"];
+
+triggers any => 'indego', 'bikeshare', 'bike share', 'ride indego', 'share bike';
+
+spice to => 'https://api.phila.gov/bike-share-stations/v1';
+spice wrap_jsonp_callback => 1;
+spice proxy_cache_valid => '200 304 15m';
+
+handle remainder => sub {
+    $loc->city.' '.$_ =~ /(philadelphia|phl|philly)/i;
```

Comment:
We need to check that `$loc` and `$loc->city` are defined before using them: ` return unless $loc and $loc->city;`